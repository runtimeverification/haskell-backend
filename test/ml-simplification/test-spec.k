module TEST-SPEC
    imports TEST

    claim <k> runTest( X ) => .K </k>
        requires {X +Int 5 #Equals Y +Int 7}

endmodule
