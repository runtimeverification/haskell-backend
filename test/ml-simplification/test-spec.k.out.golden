  <k>
    doneTest ( X:Int ~> . ) ~> .
  </k>
#And
  {
    X +Int 5
  #Equals
    Y +Int 7
  }
#And
  {
    true
  #Equals
    #Top
  }
